app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -111,8 +145,10 @@
 </template>
 
 <script>
+import baseModal from '../base/base-modal.vue';
 
 export default {
+  components: { baseModal },
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  The "baseModal" component has been registered but not used        vue/no-unused-components